ETL/ETL/_clock_system.h

```diff
@@ -29,25 +29,11 @@
 
 /* === H E A D E R S ======================================================= */
 
-#ifndef _WIN32
-# include <time.h>
-# define __sys_clock	::clock
-# define __sys_time	::time
-#else
-# ifdef __GNUG__
-#  include <time.h>
-#  define __sys_clock	::clock
-#  define __sys_time	::time
-# else
-typedef int clock_t;
-typedef int time_t;
-extern clock_t _clock();
-extern time_t _time(time_t *);
```

Comment:
I can't find those two functions above anywhere in synfig code, but here. I used this command line.
`$ git grep \\b_time\s*\(`
So I think this fallback was actually never reached and it's fine to delete.

Reply:
I think i was remove it :)
Am I miss something?

Reply:
I just meant I agree :)